layers/+lang/latex/packages.el

```diff
@@ -77,10 +77,16 @@
         "mv" 'TeX-view))))
 
 (when (string= latex-build-command "LatexMk")
-(defun latex/init-auctex-latexmk ()
-  (use-package auctex-latexmk
-    :defer t
-    :init (add-hook 'LaTeX-mode-hook 'auctex-latexmk-setup))))
+  (defun latex/init-auctex-latexmk ()
+    (use-package auctex-latexmk
+      :defer t
+      :init
+      (progn
+        (add-hook 'LaTeX-mode-hook
+                  (lambda()
+                    (setq TeX-command-default "LatexMk")))
+        (auctex-latexmk-setup)
```

Comment:
Ah I see. Well, `auctex-latexmk-setup` only needs to be called once, it certainly doesn't need to be in `LaTeX-mode-hook`. Based on my very basic understanding of use-package this should be called in a `:config` macro. Is that a correct way to do it that works with lazy-loading?